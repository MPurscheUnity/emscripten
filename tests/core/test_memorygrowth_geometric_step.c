#include <stdio.h>
#include <stdlib.h>

#include <emscripten/heap.h>

int main()
{
	malloc(60*1024*1024);
	printf("Heap size: %zu\n", emscripten_get_heap_size());
}
