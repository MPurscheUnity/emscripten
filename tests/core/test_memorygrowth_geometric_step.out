Heap size: 268435456
